index.bs

```diff
@@ -618,6 +620,14 @@ This algorithm will return either a non-negative 128-bit integer or an error.
 1. If the characters within |value| are not all [=ASCII hex digits=], return an error.
 1. Interpret |value| as a hexadecimal number and return as a non-negative 128-bit integer.
 
+<h3 dfn id="can-attribution-rate-limit-record-be-removed">Can attribution rate-limit record be removed</h3>
+
+Given an [=attribution rate-limit record=] |record|:
+1. If |record|'s [=attribution rate-limit record|time=] is after the current time, return false.
+1. If |record|'s [=attribution rate-limit record/scope=] equals `"attribution"`, return true.
```

Comment:
Just double checking that this is intentionally `true`, not `false`.

Reply:
Thanks Andrew. Yes, it's intentionally true as for attribution we just need to check the time field.